internal/notification/notification.go

```diff
@@ -0,0 +1,35 @@
+package notification
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"gocloud.dev/pubsub"
+)
+
+type pkgIdentifier struct {
+	Name      string
+	Version   string
+	Ecosystem string
+}
+
+type NotificationMessage struct {
```

Comment:
Can this type be unexported?

Reply:
Something will be consuming this very soon right?

Reply:
Ah true, in that case disregard